test/test_torch.py

```diff
@@ -3208,6 +3208,13 @@ def test_logical_ops(self):
         x_clone ^= y
         self.assertEqual(x_clone, xor_result)
 
+    def test_issue_1549(self):
```

Comment:
Can you remove that test? It'd be better to add a more general one that would compare outputs of each bitwise function to manual application of these operators to Python scalars taken elementwise.